src/App.js

```diff
@@ -1,9 +1,54 @@
-import './App.css';
+import { useState, useEffect } from "react";
+import "./App.css";
+import List from "./List";
+import EditMemoPanel from "./EditMemoPanel";
+
+export default function App() {
+  const [isEditing, setIsEditing] = useState(false);
+  const [memos, setMemos] = useState(() => {
+    const memoList = [];
+    for (let i = 0; i < localStorage.length; i += 1) {
```

Comment:
localStorageにメモしか入っていないことが前提になっているようですが、localStorageというのは色々な用途で使われるものです。メモ以外のデータも格納されることがあるという前提に立ったほうが良いです。

まずkeyを見てそこに何のデータが入っているのか分かるようなkey nameにしましょう。
その上で必要なkeyに対してのみ処理を行うべきです。

Reply:
すみません、https://github.com/GennyBoy/react-memo-spa/pull/1#discussion_r1657557827 とほぼ同じようなコメントを書いてしまいました 😓 
どちらか片方は消したつもりだったのですがうまく消せていなかったようです。
言いたいことは同じなので、localStorageの再設計をお願いしますー

Reply:
こちらもご指摘ありがとうございます！

下記のように、Prefix に `memo-` とついているものだけに対して処理するようにいたしました。
https://github.com/GennyBoy/react-memo-spa/blob/f1c21d52cca1f55c9671e9cac1b8f65b4e9e1184/src/App.js#L16-L25

Reply:
これだと根本的には変わっていないというか、LocalStorageに格納されている全てのkeyを順番にチェックしていくという処理のままですね。memoの他にもたくさんkeyがあったりすると効率が良くないので、必要なkeyだけを取得するような処理（とデータ設計）にしてください。

また、`localStorage.key()`を使って上から順番に取得していく処理だと、メモの順番がブラウザの実装依存になってしまうのであまり良くない気がします。（順番を気にしないのならいいですが、流石にそういうわけにはいかないかなと）
参考： https://developer.mozilla.org/en-US/docs/Web/API/Storage/key#:~:text=The%20order%20of%20keys%20is%20user%2Dagent%20defined%2C%20so%20you%20should%20not%20rely%20on%20it.

Reply:
というか、今やってみたら、メモが複数ある状態で新規メモを登録してからリロードすると順番が変わる時がありますね。ちょっと不自然な動きだと感じました。
(おそらく、Chromeはキーを辞書順にソートしているからだと思います）

Reply:
あと、これは本質的なところではないですが、普通はこう書くと思います。
```suggestion
    for (let i = 0; i < localStorage.length; i++) {
```